### DISCUSSION TITLE
No documentation on jamulus.io

### DISCUSSION THREAD

**Discussion by hgbork** [jamulussoftware/jamulus#1541] [Documentation]
Hi,

all documentation links on jamulus.io lead to a 404 error. Same applies to the wiki pages on https://github.com/jamulussoftware/jamulus/wiki
Is this a general problem or are the pages moved somewhere else?

With kind regards
-- hgb

**Comment by ann0see** [comment]
Just pushed something. https://jamulus.io/index.html seems to work?

**Reply by ann0see** [reply]
Ok. Just force-pushed and reverted to the old page. I'll try to debug it.

**Reply by ann0see** [reply]
Ok. Not sure what went wrong, but now it seems to work

**Reply by hgbork** [reply]
Yep. Works again. Thanks

**Reply by ann0see** [reply]
Great!
